### PR TITLE
Add channels to publish a Tweet on

### LINKED ISSUE #1424: Define channels to publish for a Tweet
- Issue type: Feature request

### Short description

In #1414, publish hookks were added to `Post` and now we can have a method which defines the channels to pubish a Tweet to for streaming.

### PR DESCRIPTION
**Pull request by singhpratyush**
### Short description

Fixes #1424.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by singhpratyush** [comment]
@Achint08 @djmgit @hemantjadon @kavithaenair @SKrPl @vibhcool: Please review.

File: src/org/loklak/harvester/TwitterScraper.java [hunk 1]
```diff
@@ -26,25 +26,17 @@
...
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```
Comment (vibhcool) [inline src/org/loklak/harvester/TwitterScraper.java]: importing selected utils would be better

Comment (singhpratyush) [inline src/org/loklak/harvester/TwitterScraper.java]: All of them are used in the file.

Comment (vibhcool) [inline src/org/loklak/harvester/TwitterScraper.java]: see this https://docs.oracle.com/javase/7/docs/api/java/util/package-summary.html

Comment (singhpratyush) [inline src/org/loklak/harvester/TwitterScraper.java]: Though we are not following those guidelines, I have made the changes. Please take a look.

**Pull request merged**